src/aideck/embeddings/azure_openai_provider.py

```
@@ -0,0 +1,449 @@
+            
+            return embeddings
+            
+        except openai.RateLimitError as e:
+            wait_time = 60
+            if "retry-after" in str(e):
+                try:
+                    import re
+                    wait_match = re.search(r'retry-after[:\s]+(\d+)', str(e), re.IGNORECASE)
+                    if wait_match:
+                        wait_time = int(wait_match.group(1))
+                except:
```

Ruff (0.13.1)

Do not use bare `except`

(E722)